### PR TITLE
Starter blog

### LINKED ISSUE #1615: Blog about the Open Liberty Starter
Let's write a blog on yet another way to get started on Open Liberty. We should discuss the starter in the first half and then summarize other ways someone can get started with Open Liberty like using a docker image, maven/gradle guides, downloading the binary, etc.

### PR DESCRIPTION
**Pull request by steven1046**

### CODE REVIEW

File: posts/2021-08-31-open-liberty-starter.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[21.0.0.8,)'
+}
+----
+
+For more information on using Open Liberty and Gradle, visit our link:{url-prefix}/guides/gradle-intro.html[Gradle Intro Guide].
+
+=== Open Liberty Binary
+
+Download an Open Liberty release from our link:Getting Started[{url-prefix}/downloads/#runtime_releases_table_container] page. Then navigate to the `bin` directory and execute `./server create <server_name>` to create an Open Liberty server. Then, you can add in your application to the dropins directory, or add the path to your application in the newly created `<path_to_Open_Liberty_install>/servers/<server_name>/server.xml`.
```
Comment (mbroz2) [inline posts/2021-08-31-open-liberty-starter.adoc]: Add something regarding unzip/extract:
Download an Open Liberty release from our link:Getting Started[{url-prefix}/downloads/#runtime_releases_table_container] page **and extract the package.**

Comment (mbroz2) [reply posts/2021-08-31-open-liberty-starter.adoc]: Then navigate to the `bin` directory and execute `./server create <server_name>` to create an Open Liberty server, or use `./server start` to automatically create and start a `defaultServer`.

Comment (mbroz2) [reply posts/2021-08-31-open-liberty-starter.adoc]: Add links to docs, something like:
For more information regarding Liberty commands and configuration, visit the link:Command overview[{url-prefix}/docs/latest/reference/command/command-overview.html] and link:Server configuration overview[{url-prefix}/docs/latest/reference/config/server-configuration-overview.html] docs.

File: posts/2021-08-31-open-liberty-starter.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "Get started using the Open Liberty Starter"
```
Comment (mbroz2) [inline posts/2021-08-31-open-liberty-starter.adoc]: I think we can improve on the title.... maybe "Introducing the Open Liberty Starter!" or something else.... can you ask @dmuelle or @gcharters for suggestions?

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: posts/2021-08-31-open-liberty-starter.adoc (hunk 2 above)
Comment (steven1046) [reply posts/2021-08-31-open-liberty-starter.adoc]: @dmuelle and @gcharters, do you have any advice on the title?

File: posts/2021-08-31-open-liberty-starter.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Steven Zvonek <https://github.com/steven1046>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+== Open Liberty Starter Application
+
+There is now another easy way to get started using Open Liberty. Visit the link:{url-prefix}/downloads/[Getting Started] page and select a few options to generate a starter application using Open Liberty. You can specify your application and project name, choose a build tool from either Maven or Gradle, and pick which version of Java SE, Jakarta EE, and MicroProfile your application will use. Then, just click `Generate Project` and you are ready for lift-off!
```
Comment (mbroz2) [inline posts/2021-08-31-open-liberty-starter.adoc:23]: I think we should add another paragraph that explains what to do once they download the package (unzip & what command(s) to run).  (Really, I think this should be included with the starter, on the site and/or a README in the zip.  
Lastly, maybe provide links to  (https://openliberty.io/guides/?search=microprofile&key=tag) and Jakarta EE guides (https://openliberty.io/guides/?search=jakarta%20ee&key=tag) for further info on those.

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: posts/2021-08-31-open-liberty-starter.adoc (hunk 3 above)
Comment (steven1046) [reply posts/2021-08-31-open-liberty-starter.adoc:23]: We do have https://github.com/OpenLiberty/start.openliberty.io/issues/9, so i could add some of the commands in there and briefly talk about it in the blog.

Comment (mbroz2) [reply posts/2021-08-31-open-liberty-starter.adoc:23]: I have no issues with us adding extra instructions to the README.md of start.openliberty.io repo, but that should not be the main way that users obtain that information nor should that be used in the blog.  For the blog, just directly provide the information (which you can then also paste into the readme.md of that repo.)

Long-term, we should update the user flow to provide them with this information.  For example, once the user clicks "Generate Projects" on /downloads, we could then show them the commands based on which build project they selected (and we could also include those commands in some sort of file in the zip, again, pertaining to the build system they selected).  I've added this to the agenda for our ol.io call.

**Comment by steven1046** [comment]
@mbroz2 i have updated the title and added some more instructions on what to run after downloading the starter.

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**